demo/loader/entries/boot.conf

```diff
@@ -0,0 +1,4 @@
+title       Grisp2 demo
```

Comment:
Can we set the title to `GRiSP 2 Demo`?

In the Erlang loader entry config we also have:
```
architecture ARM
```

I think it shows up in the boot menu.

Reply:
Fixed